runtime/devnet/src/config/assets.rs

```diff
@@ -30,6 +30,7 @@ parameter_types! {
 parameter_types! {
 	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
 	pub const NftsCollectionDeposit: Balance = 10 * UNIT;
+	pub const NftsCollectionApprovalDeposit: Balance = deposit(1, 0);
```

Comment:
Why arent we specifying the number of bytes? Seems that (1,0) is only used for a base deposit, with per byte deposits still being added.

Reply:
Resolved: https://github.com/r0gue-io/pop-node/pull/387#:~:text=CollectionApprovalDeposit%3A%20The%20basic%20amount%20of%20funds%20that%20must%20be%20reserved%20for%20collection%20approvals.